### PR TITLE
Log extensions that were loaded

### PR DESCRIPTION
**Pull request by twsouthwick**

### CODE REVIEW

File: src/cli/Microsoft.DotNet.UpgradeAssistant.Cli/Program.cs [hunk 1]
```diff
@@ -115,9 +115,9 @@ private static IHostBuilder EnableLogging(UpgradeOptions options, IHostBuilder h
 
                     services.AddSingleton<IUpgradeStateManager, FileUpgradeStateFactory>();
 
+                    services.AddExtensions(context.Configuration, options.Extension);
```
Comment (mjrousos) [inline src/cli/Microsoft.DotNet.UpgradeAssistant.Cli/Program.cs:118]: Does the order here matter? I would've thought it didn't.

File: src/components/Microsoft.DotNet.UpgradeAssistant.Extensions/ExtensionProviderExtensions.cs [hunk 2]
```diff
@@ -8,8 +8,12 @@
 using System.Runtime.Loader;
 using System.Text.Json;
 using System.Text.Json.Serialization;
+using System.Threading;
```
Comment (mjrousos) [inline src/components/Microsoft.DotNet.UpgradeAssistant.Extensions/ExtensionProviderExtensions.cs:11]: The `usings` updates here don't look necesary.

**Review by mjrousos — APPROVED** [review]
LGTM

File: src/cli/Microsoft.DotNet.UpgradeAssistant.Cli/Program.cs (hunk 1 above)
Comment (twsouthwick) [reply src/cli/Microsoft.DotNet.UpgradeAssistant.Cli/Program.cs:118]: Just so the messages of extensions are shown first

**Pull request merged**